tests/unit/core/test_addonManager.js

```diff
@@ -221,7 +221,8 @@ describe(__filename, () => {
       });
 
       sinon.assert.calledWith(fakeMozAddonManager.createInstall, {
-        url,
```

Comment:
The test cases I fixed here are obsolete-ish. All we need is to fix https://github.com/mozilla/addons-frontend/issues/9556 so ignore the changes here 😬